src/sql/parts/common/customInputConverter.ts

```diff
@@ -129,6 +141,29 @@ function getQueryPlanEditorUri(input: EditorInput): URI {
 	return undefined;
 }
 
+
+/**
+ * If input is a supported notebook editor file (.ipynb), return it's URI. Otherwise return undefined.
+ * @param input The EditorInput to get the URI of
```

Comment:
please finish the comment

Reply:
The comment says, input is the type of EditorInput object to get the URI of. It's a generic class which we are using it for Notebook scenario. We can change it in a meaningful way. I just copied it from the older method. Did not change the description as it made sense to me.